_pages/about/officers.md

```diff
@@ -3,6 +3,13 @@
     title: Club Officers
 ---
 
+## Become a Club Officer
+
+Interested in become a club officer? Join the Board of Directors (BOD)! We are a group of members and volunteers who all care deeply about the club and enabling the MITOC community to enjoy being outside. We do everything from organizing Winter School, maintaining our gear and cabins, to recruiting new leaders and members.
```

Comment:
I think technically the Board of Directors are only the voting members, and the rest of the club officers are the "Advisory Board".

Reply:
eh, according to the bylaws there is no such thing as a board of directors, it's the "Executive Board" (voting) and "Advisory Board". I just stole this from the other page, if someone wants to change it they should do across the site. Leaving as-is.